### PR TITLE
remove reference to "club"

### PR DESCRIPTION
**Pull request by moneyball**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ If you sign up prior to Oct 30th (23:59 UTC-10) we’ll help you find other peop
 * The final Q&A sessions can be used to raise any concerns that haven’t already been covered, or discuss some of the proof of concept work
 * And finally, on Thursday or Friday, you should complete your review and fill in the google form, and, if you have any concerns that haven’t already been addressed, work on ensuring that’s raised as an issue or pull request against the relevant BIPs.
 
-## What happens this review club is over
+## What happens this review is over
```
Comment (jnewbery) [inline README.md]:
```suggestion
## What happens when this review is over
```

**Comment by jnewbery** [comment]
ACK modulo nit

File: README.md (hunk 1 above)
Comment (moneyball) [reply README.md]: Thanks. Addressed this by rephrasing it altogether.

**Comment by jnewbery** [comment]
I preferred "when X is over" to "at the completion of X" but sure. ACK 2156e4f5c13d544150df1a28ec3a2b9e52c67f01

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ If you sign up prior to Oct 30th (23:59 UTC-10) we’ll help you find other peop
 * The final Q&A sessions can be used to raise any concerns that haven’t already been covered, or discuss some of the proof of concept work
 * And finally, on Thursday or Friday, you should complete your review and fill in the google form, and, if you have any concerns that haven’t already been addressed, work on ensuring that’s raised as an issue or pull request against the relevant BIPs.
 
-## What happens this review club is over
+## What happens at the completion of the Taproot BIP review
```
Comment (jonatack) [inline README.md:53]: perhaps s/at the/on/ ... or more simply, s/What happens at the/On/

**Review by jonatack — COMMENTED** [review]
ACK 2156e4f5c13d544150df1a28ec3a2b9e52c67f01 modulo nit

**Comment by ajtowns** [comment]
made it "after this is all over"

**Pull request closed without merging**